File: Android/PacelableStudy/app/src/main/java/com/android/gold/pacelablestudy/MainActivity.java
package com.android.gold.pacelablestudy;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void goToActivity(View v){
        Intent intent = new Intent(this, SubActivity.class);
        Student student = new Student("홍길동", 3, 17, 1033337777);
        intent.putExtra("student", student);
        startActivity(intent);
    }


}

File: Android/TaskExample/app/src/main/java/com/example/gold/taskexample/MainActivity.kt
package com.example.gold.taskexample

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btn_go_b.setOnClickListener {
            val intent = Intent(this, BActivity::class.java)
            startActivity(intent)
        }

    }

    override fun onNewIntent(intent: Intent?) {
        super.onNewIntent(intent)
        println(Log.d("TaskTest", "A_Acitivity"))
    }
}
